packages/rum-core/test/performance-monitoring/transaction-service.spec.js

```diff
@@ -167,42 +174,22 @@ describe('TransactionService', function() {
     tr2.detectFinish()
   })
 
-  it('should sendPageLoadMetrics', function(done) {
-    config.set('active', true)
-    config.set('capturePageLoad', true)
-
-    transactionService = new TransactionService(logger, config)
-
-    transactionService.subscribe(function() {
-      expect(tr.isHardNavigation).toBe(true)
-      expect(tr.marks.agent).toBeDefined()
-      expect(tr.marks.navigationTiming).toBeDefined()
-      done()
-    })
-    var tr = transactionService.sendPageLoadMetrics('test')
-
-    transactionService = new TransactionService(logger, config)
-    var zoneTr = new Transaction('ZoneTransaction', 'zone-transaction')
-    transactionService.setCurrentTransaction(zoneTr)
-
-    var pageLoadTr = transactionService.sendPageLoadMetrics('new tr')
-
-    expect(pageLoadTr).toBe(zoneTr)
-  })
-
-  it('should consider initial page load name or use location.pathname', function() {
+  it('should use initial page load name before ending the transaction', function(done) {
     transactionService = new TransactionService(logger, config)
-    var tr
 
-    tr = transactionService.sendPageLoadMetrics()
+    const tr = transactionService.startTransaction(null, 'page-load')
```

Comment:
Maybe use undefined instead of null to be the same as the old test!

Reply:
startTransaction doesn't differentiate between null/undefined. Any specific reason?